bcipy/orchestrator/parameters/actions.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+from bcipy.task import TaskType
+"""This file can define actions that can happen in a session orchestrator visit. To start these will be 1:1 with tasks, but later this can be extended to represent training sequences, GUI popups etc."""
+
+taskname_dict = {}
+for i, task in enumerate(TaskType.list()):
+    taskname_dict[task] = TaskType(i + 1)
+
+
+def parse_actions(action_sequence: str) -> List[TaskType]:
```

Comment:
As discussed in our meeting, Tasks and Actions are not going to be the same thing, so this name should change. It could just be `parse` and the parameter name `sequence`. The return type would need to a be a union of TaskType and Action.